src/DependencyInjection/Configuration.php

```diff
@@ -55,6 +55,17 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->defaultValue(['json'])
                     ->prototype('scalar')->end()
                 ->end()
+                ->arrayNode('html_config')
+                    ->info('SwaggerUI/Redocly configuration options')
```

Comment:
More generic description. Furthermore this section should also include the `assets_mode` config option. https://github.com/nelmio/NelmioApiDocBundle/blob/a585e502725d7e3cfcec25f7a3d59443842006da/src/Render/Html/HtmlOpenApiRenderer.php#L43
```suggestion
                    ->info('UI configuration options')
```

Reply:
Done